src/Nri/Ui/RadioButton/V2.elm

```diff
@@ -177,7 +178,7 @@ internalView config =
                 , ( "Nri-RadioButton-RadioButtonChecked", isChecked )
                 ]
             , css
-                [ padding4 (px 4) zero (px 4) (px 40)
+                [ padding4 zero zero zero (px 35)
```

Comment:
We're seeing a similar problem with the focus ring as the checkbox. Now, you can't see the focus ring for radio buttons at all.